components/shapediver/parameter/ParameterSliderComponent.tsx

```diff
@@ -129,6 +136,30 @@ export default function ParameterSliderComponent(
 	);
 
 	const valueClamped = roundAndClamp(+value);
+	const handleChangeCustom = useCallback(
+		(v: number | string, callback?: () => void) => {
+			const roundedValue = roundAndClamp(+v);
+			setParameterValueDebounced(roundedValue, 500, {
```

Comment:
@LemToUp only comment: Do we need to use 500 here, or can we change to undefined? (which will result in a different timeout depending on whether the parameter is using acceptRejectMode)